play.py

```diff
@@ -204,21 +205,25 @@ def play_aidungeon_2():
 
         while story_manager.story is None:
             print("\n\n")
+            change_config = input("Would you like to enter a new parameters now? (default: temp:  0.4, top_p: 0.9, cut_actions: true) (y/N) ")
+            if change_config.lower() == "y":
+                story_manager.generator.change_temp(float(input("Enter a new temp (default 0.4): ") or 0.4))
+                story_manager.generator.change_top_p(float(input("Enter a new top_p (default 0.9): ") or 0.9))
+                story_manager.generator.change_cut_actions(bool(int(input("Enter 1 to trim actions from output, 0 to preserve as much as possible (default 1): ") or "1")))
+                console_print("Please wait while the AI model is regenerated...")
+                story_manager.generator.gen_output()
+

```

Comment:
I moved this before selecting load/new because it wasn't possible to select parameters when loading a game before.

Reply:
and those parameters aren't saved or loaded currently.

Reply:
on second thought I reverted this one since it doesn't really belong in this PR.